app/src/main/java/com/framgia/englishconversation/screen/createcomment/CreateCommentViewModel.java

```diff
@@ -351,6 +359,12 @@ private void postLiteralComment(MediaModel model) {
         comment.setMediaModel(model);
         comment.setContent(mInputtedComment);
         comment.setCreateUser(mUserModel);
+         StatusModel statusCommentModel = new StatusModel();
```

Comment:
ERROR: 'method def' child have incorrect indentation level 9, expected level should be 8.